src/commands/start.js

```diff
@@ -17,35 +17,47 @@ const logger = require('winston');
 const configModule = require('../config/config');
 const repeater = require('../repeater/repeater');
 const sendHeartbeat = require('../heartbeat/heartbeat').sendHeartbeat;
-const registryFile = require('../constants').registryLocation;
-const registryFileUtils = require('../utils/registryFileUtils');
-
+const request = require('superagent');
+const httpUtils = require('../utils/httpUtils');
+const errors = require('../errors');
 /**
  * The "start" command creates the heartbeat repeater.
  *
  * @throws TODO
  */
-function execute(name) {
+function execute(collectorName, refocusUrl, accessToken) {
   debug('Entered start.execute');
   configModule.setRegistry();
   const config = configModule.getConfig();
-  const regObj = registryFileUtils.getRefocusInstance(name, registryFile);
+  config.collectorConfig.collectorName = collectorName;
+  config.collectorConfig.refocusUrl = refocusUrl;
 
-  /*
-   * TODO: Replace the success/failure/progress listeners here with proper
-   * logging once we have heartbeat
-   */
-  repeater.create({
-    name: 'Heartbeat',
-    interval: config.collectorConfig.heartbeatInterval,
-    func: () => sendHeartbeat(regObj),
-    onSuccess: debug,
-    onFailure: debug,
-    onProgress: debug,
-  });
+  const path = `/v1/collectors/${collectorName}/start`;
+  const url = refocusUrl + path;
+  return request.post(url)
+  .set('Authorization', accessToken)
```

Comment:
@harshkothari410 I tried moving this to httpUtils like you requested, but I think it's better to just do it here, since there's nothing else to be done here except starting the repeater, which is dependent on the post request. Ok?

Reply:
It's ok.